tasks/main.yml

```diff
@@ -2,6 +2,12 @@
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: "Include variables for Amazon Linux. {{ ansible_distribution }}  {{ ansible_distribution_major_version }}"
+  include_vars: "AmazonLinux.yml"
+  when:
+    - ansible_distribution == "Amazon"
+    - ansible_distribution_major_version == "NA" or ansible_distribution_major_version == "2018" or ansible_distribution_major_version == "6"
```

Comment:
Do we also need these things here, or can we just have:

```
when: ansible_distribution == "Amazon"
```

Reply:
The gig paying for this is using Amazon linux v1.

Reply:
So I guess my answer is: "Not sure"

Reply:
I'd say if it works in the generic way let's leave it (and assume it works on Amazon Linux 2 until proven otherwise!). Also, I technically have an az linux 2 base image for ansible, so if you would like to add a test case in the `.travis.yml` file then the CI runner will make sure it works (or not) :)

https://github.com/geerlingguy/docker-amazonlinux2-ansible